funnel/models/project.py

```diff
@@ -44,9 +44,13 @@ class Project(BaseScopedNameMixin, db.Model):
     tagline = db.Column(db.Unicode(250), nullable=False)
     description = MarkdownColumn('description', default=u'', nullable=False)
     instructions = MarkdownColumn('instructions', default=u'', nullable=True)
+
     datelocation = db.Column(db.Unicode(50), default=u'', nullable=False)
+    location = db.Column(db.Unicode(50), default=u'', nullable=True)
```

Comment:
Just occurred to me that this conflicts with the Venue field. We may not even need a location field.

Reply:
Also, why 50 chars?